### PR TITLE
Docs: clean up some links, navigation

### PR DESCRIPTION
**Pull request by khawkins98**
Today I am going to try and work through some paper cuts that have been bugging me.

### CODE REVIEW

**Comment by khawkins98** [comment]
This one I think is ready:

- Minor typos and formatting
- Fixes active trails in the component library's global navigation
- Fixes a markdown templating error
- Updates vf-search-client-side to match v2.0.0 of vf-search--inline
- Adds the search box back to the homepage (I keep seeing users missing it when screen sharing)

image

File: components/vf-search-client-side/vf-search-client-side.njk
Comment (sturobson) [inline components/vf-search-client-side/vf-search-client-side.njk:16]: I spotted this in the VF today - and it felt like I implemented something 'half baked'. So this might change :D

**Review by sturobson — APPROVED** [review]

**Comment by sturobson** [comment]
@khawkins98 I see there's some post approval changes - so I'll let you merge this in when you're happy it's ready.

**Pull request merged**
